assets/constants.ts

```diff
@@ -7092,6 +7092,6 @@ export default {
   ],
   "CONTRIBUTOR_CERTIFICATE_WIDTH": 1493,
   "CONTRIBUTOR_CERTIFICATE_HEIGHT": 1313,
-  "BRANCH_NAME": "",
-  "SHORT_COMMIT_HASH": ""
+  "BRANCH_NAME": "contact_us-page",
```

Comment:
This shouldn't be part of the PR changes.

Reply:
This is automatically added when making a new Branch.